test_coverage/fastapi_testing.py

```diff
@@ -0,0 +1,93 @@
+from fastapi.testclient import TestClient
+from app.api import API
+client = TestClient(API)
+
+
+def test_version():
+    response = client.get("/version")
+    assert response.status_code == 200
+    assert response.json() == {'result': '0.43.7'}, 'Versions do not match'
```

Comment:
Version here and in api_testing are both failing because the minor version changed. Maybe just check for the major version instead.